tests/test_client.py

```diff
@@ -86,7 +86,7 @@ def test_user_track_request_exception(self):
 
         response = self.client.user_track(attributes=attributes, events=events, purchases=purchases)
 
-        self.assertEqual('https://api.appboy.com/users/track', self.client.request_url)
+        self.assertEqual(self.client.API_URL + '/users/track', self.client.request_url)
```

Comment:
NB: Would sandbox urls be like `https://api.appboy.com/` as well with a different api key? if not, may need some monkey-patching when using the client against sandbox api urls.

Reply:
Perhaps we can make API_URL a part of the client constructor and default it to this value so the user of the client class can initialize it with the appropriate base api url.